src/components/ec-modal/ec-modal.vue

```diff
@@ -188,7 +188,9 @@ function getFocusTrapOptions(): UseFocusTrapOptions {
   return options;
 }
 
-const { deactivate } = useFocusTrap(focusTrapTarget, getFocusTrapOptions());
+const { deactivate, activate } = useFocusTrap(focusTrapTarget, getFocusTrapOptions());
+const focusTrapInterval = setInterval(() => activate(), 300);
```

Comment:
So this keeps activating focustrap every 300ms, is this the best way?

Reply:
Yes it keeps activating, I haven't found another way to trap the focus on modal and don't lose it if another component want's to trap it.